cclib/parser/gaussianparser.py

```diff
@@ -314,6 +314,82 @@ def extract(self, inputfile, line):
+            
+            # Keep looking until dashed only.
+            while set(line.strip()) != set("-"):
+                line = next(inputfile)
+                
+                # PCM has a few different subtypes.
+                # Model                : PCM.
+                if "Model" in line:
+                    self.metadata['solvent_model'] = " ".join(line.split()[2:])[:-1]
+                
+                # Don't think this is necessary...    
+                # elif "Atomic radii" in line:
+                #    self.metadata['solvent_params']['radii'] = " ".join(line.split()[3:])[:-1]
+                elif "Atomic radii" in line and line.split()[-1] == "SMD-Coulomb.":
+                    self.metadata['solvent_model'] = "SMD"
+                    
+                # Solvent by keyword.
+                #  Solvent              : Toluene, Eps=   2.374100 Eps(inf)=   2.238315
+                # Solvent by definition.
+                #  Solvent              : Generic,
+                #            Eps                           =   9.000000
+                #            Eps(infinity)                 =   2.000000
+
+                elif "Solvent" in line and "Eps=" in line and "Eps(inf)= " in line:
+                    # Capture the human readable name, as well as params.
+                    self.metadata['solvent_name'] = line.split()[2][:-1]
+                    
+                    self.metadata['solvent_params']['epsilon'] = float(line.split()[4])
+                    self.metadata['solvent_params']['epsilon_infinite'] = float(line.split()[6])
+                    
+                    # If flat is preferable.
+                    #self.metadata['solvent_epsilon'] = float(line.split()[4])
+                    #self.metadata['solvent_epsilon_infinite'] = float(line.split()[6])
+                
+                elif "Eps(infinity)" in line:
+                    # Assume manually specified solvent.
+                    self.metadata['solvent_params']['epsilon_infinite'] = float(line.split()[-1])
+                    
+                elif "Eps" in line:
+                    # Assume manually specified solvent.
+                    self.metadata['solvent_params']['epsilon'] = float(line.split()[-1])
+        
+        # Other solvent models are not so easy to parse (have less info).
+        # Reaction Field using a Density IsoSurface Boundary
+        if "Reaction Field using a Density IsoSurface Boundary" in line:
+            self.metadata['solvent_model'] = "IPCM"
+        
+        #  Epsi=   78.3000 Cont =    0.0010
+        if  "Epsi=" in line and "Cont =":
+            if "solvent_params" not in self.metadata:
+                self.metadata['solvent_params'] = {}
+            
+            # PLEASE REVIEW; I've guessed the meaning of these abbreviations.
+            self.metadata['solvent_params']['epsilon'] = float(line.split()[1])
+            self.metadata['solvent_params']['continuity'] = float(line.split()[4])
```

Comment:
Is `Cont = ` contour maybe?

Reply:
Yeah, isovalue for surfaces constructed from the density.

Reply:
ty